tools/serverpod_cli/bin/create/create.dart

```diff
@@ -26,13 +26,11 @@ Future<void> performCreate(
   bool force,
 ) async {
   // Check whether the project name is valid or not
-  for (int i = 0; i < name.length; i++) {
-    int ascii = name[i].codeUnitAt(0);
-    if (!((ascii >= 97 && ascii <= 122) || name[i] == '_')) {
-      print(
-          "Project Name can only contain lowercase letters and underscores..");
-      return;
-    }
+  RegExp namePattern = new RegExp(r'^[a-z][a-z0-9_]*');
+  if (namePattern.stringMatch(name).toString().length != name.length) {
```

Comment:
This condition actually works in most cases. But we can break it with any 4 special chars. Check out the bottom sample code with your condition.

<img width="922" alt="Screenshot 2022-12-09 at 3 22 26 AM" src="https://user-images.githubusercontent.com/35523357/206575158-dab1df31-3dd8-4976-82f1-262df4aa15b8.png">

I can give you one more hint. This time let's be more specific. `^[0-9a-z]+(?:[0-9a-z]+|_[0-9a-z]+)+$` Use this regex and use the match method.

> NOTE: Before pushing code, check it out in [dartpad](https://dartpad.dev) and check the regex match with [Regex Tester](https://www.regextester.com/)